client/state/selectors/get-sites-for-upcoming-transactions.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * External dependencies
+ */
+import { map, uniq } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import { getSite } from 'state/sites/selectors';
+import { getUpcomingBillingTransactions } from 'state/selectors';
+
+const getSitesForUpcomingTransactions = createSelector(
```

Comment:
JSDoc added in a2945c86a8338e97a050fdcd4acbb7a6c267c0ba.
Unit tests added in e050d365a6e6f98186c5fa813a6551fb09e2b272.